accumulator/readme.md

```diff
@@ -1,28 +1,25 @@
 A dynamic hash-based accumulator.
 
-Overview
---------
+## Overview
```

Comment:
There's no need for these changes since github renders the existing ones just fine right? (or are there IDEs that don't render these right?)

Reply:
My code editor autoformatted it to a `##`, will revert it back in my next commit.